### ISSUE TITLE
[FeatureRequest] Add EAST text detection support

### ISSUE THREAD

**Issue by rcd27** [JetBrains/KotlinDL#336]
There is very usefull pretrained Tensorflow model for detecting text on image: https://github.com/argman/EAST

Would be nice to use that in KotlinDL

**Comment by rcd27** [comment]
This is a `Kotlin` implementation with use of `OpenCV.DNN` modulde + `EAST` detection model:
https://github.com/rcd27/opencv-EAST-kotlin

**Comment by rcd27** [comment]
Yes, the model format is `.pb`, I'll try to go through the steps you suggested.

**Comment by rcd27** [comment]
@zaleslaw all EAST models with `.pb` format I found in the internet were missing `tag` value (checked with `saved_model_cli` tool). That error is thrown by `tensorflow-onnx` while trying to convert. Any ideas how to workaround this?

**Comment by zaleslaw** [comment]
Looks like it requires a tag setting before convertation
I found a few contants here
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/saved_model/tag_constants.py#L27
